### ISSUE TITLE
Mail URLs (`mailto:`) are reported as dead

### ISSUE THREAD

**Issue by wooorm** [remarkjs/remark-lint-no-dead-urls#4]
```
🚀  remark . -qfo
readme.md
  129:1-129:62  warning  Link to mailto:t.e.wormer@hva.nl?subject=frontend-3:%20 is dead     no-
⚠ 1 warning
```

We should either add an allowed list (`https:`, `http:`) or disallowed list (`mailto:`). What do you think?

**Comment by davidtheclark** [comment]
I think an allowed list, `https:` and `http:`, sounds good. We can warn in the README that only URLs with these protocols are checked.

**Comment by wooorm** [comment]
Awesome! 🎉 Thanks David!
